nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -862,7 +828,7 @@ class LsqQuantizer(Quantizer):
        https://arxiv.org/pdf/1902.08153.pdf
     """
 
-    def __init__(self, model, config_list, optimizer):
+    def __init__(self, model, config_list, optimizer, dummy_input=None):
```

Comment:
It seems we can easily support `fold_bn` by adding `dummy_input` interface? As we discussed before, maybe we need to think about how to support other kinds of pattern fusion simulation in the future. And this pr is good to me.